dashboard/app/models/levels/level.rb

```diff
@@ -874,7 +874,7 @@ def project_type
   # Whether this level has validation for the completion of student work.
   def validated?
     if uses_lab2?
-      return properties.dig('level_data', 'validations').present?
```

Comment:
Discussed [on slack](https://codedotorg.slack.com/archives/C044AGTKW3D/p1723657092363209) that this was an incorrect check after some changes by @sanchitmalhotra126. Accessing `properties['validations']` (what `get_validations` does) is the correct change.